AmericanMedicalAssociation/AMA-style-guide
# Ama 131 ribbon header responsive

https://palantir.atlassian.net/browse/AMA-131
https://palantir.atlassian.net/browse/AMA-117

To test: 

- Navigate to Organisms > Global > Ribbon
- Observe that Ribbon exists and is responsive
- Navigate to Templates > Home > Home
- Observe that Ribbon is part of Home and is responsive
- Observe adjusted vertical spacing between home page items

Notes: 
- this isn't pixel perfect, but it's a rough pass.
To do in the future: create variables for responsive spacing quantities and use them here.
